src/content/docs/ja/reference/configuration.mdx

```diff
@@ -807,6 +815,14 @@ JSON（およびその拡張言語）ファイルの行の文字数を指定し
 
 > デフォルト: `true`
 
+### `json.formatter.objectWrap`
+
+可能な場合にオブジェクトリテラルを折り畳むように矯正するかどうかを指定します。
```

Comment:
```suggestion
可能な場合にオブジェクトリテラルを折り畳むように強制するかどうかを指定します。
```

Reply:
This was a misconversion came from a08a28c914ed158689379e2df843c48851597888, thank you for noticing this 🙏